# StereoFeaturesStamped
std_msgs/Header header
uint32 image_id
Feature[] left
Feature[] right

